lib/Core/SymbolicError.h

```diff
@@ -150,13 +155,17 @@ class SymbolicError {
   void recomputePathProbability(llvm::BasicBlock *dst, llvm::BasicBlock *src) {
     if (pathProbability < 0) {
       pathProbability = EdgeProbability::instance->getEdgeProbability(dst, src);
+      branchCount++;
```

Comment:
@Himeshi Adding a single `branchCount++` at the start of this method would be more efficient than adding two `branchCount++`.